.circleci/osx_install_dependencies.sh

```diff
@@ -52,8 +52,8 @@ then
   rm -rf z3-4.8.7-x64-osx-10.14.6
 
   # evmone
-  wget https://github.com/ethereum/evmone/releases/download/v0.3.0/evmone-0.3.0-darwin-x86_64.tar.gz
-  tar xzpf evmone-0.3.0-darwin-x86_64.tar.gz -C /usr/local
-  rm -f evmone-0.3.0-darwin-x86_64.tar.gz
+  wget https://github.com/ethereum/evmone/releases/download/v0.4.0/evmone-0.4.0-darwin-x86_64.tar.gz
```

Comment:
Why does this have 0.4.0 instead of 0.4.1?

Reply:
Set back to 0.4.0 due to
https://github.com/ethereum/solidity/pull/8651#issuecomment-613322157

The idea now is to use 0.4.1 for Windows only.